tempodb/backend/raw.go

```diff
@@ -178,3 +187,7 @@ func CompactedMetaFileName(blockID uuid.UUID, tenantID string) string {
 func RootPath(blockID uuid.UUID, tenantID string) string {
 	return path.Join(tenantID, blockID.String())
 }
+
+func ShouldCache(name string) bool {
+	return name != MetaName && name != CompactedMetaName && name != BlockIndexName
```

Comment:
I would like to expose `shouldCache bool` on the Reader/Writer interfaces already, I thought it would be neat to consolidate the caching logic in one place. But if its preferred to pass true/false I'll make that change.